LiskHQ/lisk-sdk
# Move modules/migrations to modules/components/storage/migrations - Closes #4483

### What was the problem?
migrations are part of storage components for each module, so its appropriate to keep the migrations under storage component for specific modules.

### How did I solve it?
Move

- chain/migrations -> chain/components/storage/migrations
- network/migrations -> network/components/storage/migrations

### How to manually test it?
- Create a fresh database `lisk_dev`
- Run `LISK_CONSOLE_LOG_LEVEL=debug node framework/test/test_app | npx bunyan -o short`
- Verify if the migrations are actually applied
### Review checklist

- [ ] The PR resolves #4483
